src/main/java/net/sf/mzmine/util/scans/sorting/ScanSortMode.java

```diff
@@ -18,6 +18,12 @@
 
 package net.sf.mzmine.util.scans.sorting;
 
+/**
+ * Sort all scans (-> masslists -> DataPoint[]) by MAX_TIC or NUMBER_OF_SIGNALS

```

Comment:
ScanSortMode and ScanSorter and MasslistSorter

Collection of sorting modes. 
The ScanSorter always works on MassLists